src/install/mongodb/intro.mdx

```diff
@@ -0,0 +1,22 @@
+---
+headingText: Before we start
+componentType: default
+---
+
+import infrastructureMongoDashboardCropped from 'images/infrastructure_screenshot-crop_mongodb-dashboard-cropped.webp'
+
+Our MongoDB integration collects and sends metrics from your MongoDB database to our platform, where you can monitor throughput, collection size, connections, indexes, and more
+
+With this data, you can use New Relic to create alerts when throughput spikes, create dashboards to monitor collection size, and more.
```

Comment:
I saw you're working on the intro for OHI-specific. Something to consider is maybe describing the install process in a broad but concise, overviewed way. From a reader POV, they have several installations to do even before you install the integration. Could be a bit fatiguing/confusing